### PR TITLE
[Scheduler] Fix NPE in `debug:scheduler` command

### LINKED ISSUE #51052: [Scheduler] NPE in debug:scheduler command
### Symfony version(s) affected

6.3.1

### Description

https://github.com/symfony/symfony/blob/0458c9b5348289b4a45bee91d4475a59f1948b02/src/Symfony/Component/Scheduler/Command/DebugCommand.php#L120

This line causes an NPE since `getNextRunDate` can return null.

### How to reproduce

Return null from any trigger (e.g. `new CallbackTrigger(fn () => null)` and run `debug:scheduler` command

### Possible Solution

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by maxbeckers**
| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #51052
| License       | MIT
| Doc PR        | n/a

Fix NPE in `debug:scheduler` command.

### CODE REVIEW

File: src/Symfony/Component/Scheduler/Command/DebugCommand.php [hunk 1]
```diff
@@ -114,10 +114,12 @@ private static function renderRecurringMessage(RecurringMessage $recurringMessag
             $triggerName .= ": {$trigger}";
         }
 
+        $nextRun = $recurringMessage->getTrigger()->getNextRunDate(now());
+
         return [
             $messageName,
             $triggerName,
-            $recurringMessage->getTrigger()->getNextRunDate(now())->format(\DateTimeInterface::ATOM),
+            $nextRun ? $nextRun->format(\DateTimeInterface::ATOM) : '-',
```
Comment (kbond) [inline src/Symfony/Component/Scheduler/Command/DebugCommand.php]:
```suggestion
            $recurringMessage->getTrigger()->getNextRunDate(now())?->format(\DateTimeInterface::ATOM) ?? '-',
```

**Review by kbond — CHANGES_REQUESTED** [review]
Good catch.

File: src/Symfony/Component/Scheduler/Command/DebugCommand.php [hunk 2]
```diff
@@ -114,10 +114,12 @@ private static function renderRecurringMessage(RecurringMessage $recurringMessag
             $triggerName .= ": {$trigger}";
         }
 
+        $nextRun = $recurringMessage->getTrigger()->getNextRunDate(now());
+
```
Comment (kbond) [inline src/Symfony/Component/Scheduler/Command/DebugCommand.php]:
```suggestion
```

**Review by fabpot — APPROVED** [review]

**Review by kbond — APPROVED** [review]

**Comment by kbond** [comment]
Thank you @maxbeckers.

**Pull request merged**